duckduckgo/macos-browser
# wire up sync account creation and recovery to backend

Task/Issue URL: https://app.asana.com/0/414235014887631/1204209043038910/f
Tech Design URL:
CC:

**Description**:

Update BSK for Sync API changes, and minor change to UI workflow so that account is not created until the right moment.

**Steps to test this PR**:
1. See steps in https://github.com/duckduckgo/BrowserServicesKit/pull/271
2. Smoke test Sync UI - "connect" mode doesn't work yet



---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
